app.py

```diff
@@ -2,14 +2,13 @@
 from datetime import datetime
 
 from dotenv import load_dotenv
-from flask import Flask, jsonify, request
+from flask import Flask, jsonify, request, render_template
 from flask_cors import CORS
 from recommenders.popularity_recommender import recommend_popular_movies
 from recommenders.content_based import content_based_recommendation
 from recommenders.collaborative_filtering import collaborative_recommendation
 from urllib.parse import unquote, quote
```

Comment:
The conditional statement should be indented one level deeper to align with the function definition at line 9.